include/nbl/builtin/glsl/virtual_geometry/virtual_attribute_fetch.glsl

```diff
@@ -8,19 +8,173 @@
 #include <nbl/builtin/glsl/format/decode.glsl>
 #include <nbl/builtin/glsl/virtual_geometry/virtual_attribute.glsl>
 
+//TODO: R8, R8G8, R8G8B8, RGBA8, R16, RGB16
+// float nbl_glsl_VG_attribFetch_R8_UNORM(in nbl_glsl_VG_VirtualAttributePacked_t attr, in uint vertexID)
+// {
+//   return nbl_glsl_VG_attribFetch1f(attr, vertexID);
+// }
+// vec4 nbl_glsl_VG_attribFetch_RGBA8_SFLOAT(in nbl_glsl_VG_VirtualAttributePacked_t attr, in uint vertexID)
+// {
+// #ifdef _NBL_VG_USE_SSBO
+//   return unpackUnorm4x8(nbl_glsl_VG_attribFetch1u(attr, vertexID))
+// #else
+// #endif
+// }
+
+#if (defined(_NBL_VG_USE_SSBO) && defined(_NBL_VG_USE_SSBO_UVEC4)) || _NBL_VG_FLOAT_BUFFERS_COUNT
```

Comment:
roll this into a single proxy-define to improve readability

Reply:
there are lots of combinations of macros, it will not improve readability too much..